python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -2091,20 +2089,19 @@ def __setattr__(self, attr, value):
 
             # reduce duplicate values here, leave only duplicates for LCG,
             # where we can have replicas
-            uniqueValuesDict = []
-            uniqueValues = []
+            uniqueValuesDict = oDict()
```

Comment:
The full problem is this:
I have 2 `MassStorageFile` objects which are similar but request that the same file at the end of the job be placed in 2 different locations on MassStorage. This test here currently disallows this (not entirely bad) workflow. I was looking to improve the situation and discovered there is a list of keys and a list of values being used to sort everything whilst preserving the order. Given the 2 lists were being kept in sync this looked entirely like an appropriate situation for an ordered dict unless I've misunderstood what the original code is doing?